crates/wallet/src/wallet/mod.rs

```diff
@@ -1456,12 +1456,12 @@ impl Wallet {
                 required_utxos.clone(),
                 optional_utxos.clone(),
                 fee_rate,
-                outgoing.to_sat() + fee_amount.to_sat(),
+                outgoing + fee_amount,
```

Comment:
We can remove the call to `clone` above when passing the required/optional utxos to `coin_select`.

Reply:
Fixed.